static/sass/styles.scss

```diff
@@ -527,6 +530,10 @@ summary {
       color: inherit;
     }
   }
+
+  .is-muted {
```

Comment:
Have you tried with `u-text--muted`? https://vanillaframework.io/docs/base/typography#muted-text